### ISSUE TITLE
JavaScript (Beta) - Intermediate OOP - Platformer Game (Feedback)

### ISSUE THREAD

**Issue by intherightdirection** [freeCodeCamp/freeCodeCamp#53241]
## [Step 79](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-intermediate-oop-by-building-a-platformer-game/step-79)

The instructions are really confusing and could be rephrased.

![Pasted image 20240116112831](https://github.com/freeCodeCamp/freeCodeCamp/assets/28659609/4bc422b0-1c28-4170-bba5-ef254eb8c385)

I don't think what is referred to as a conditional is a what people are going to think of as a conditional. It is just a Boolean expression that is being asked for. No action or assignment is being given depending on truthiness of the expression.

It seems like it is looking for an if, switch, ternary or bare conditional, and I believe only ternary or a bare conditional can be placed inside of an array (Not being nested inside of IIFE) which is knowledge a novice probably won't have.

The difference, although small, is that a condition takes an action based on the result of a Boolean expression. No conditional action is being taken, so it is just an expression.

Might ask for the first element in the array to be a Boolean expression of the sum of the player's `y` position, height and `y` velocity greater than or equal to the platform's `y` position. 

There are various steps that follow that refer to Boolean expressions as conditionals. The concern is people learning incorrect terminology and getting mixed up in the field.

**Comment by jdwilkin4** [comment]
Yes, you are correct we should update the terminology for steps 78-81 to say boolean expression instead of conditional statement. 

I have opened this up for contribution.

**Comment by jdwilkin4** [comment]
## Note to any interested contributors

For each of the steps, make sure to make the change mentioned above in all of the descriptions.
Also, go through and check all of the hint text to make sure the correct terminology is used as well

**Comment by intherightdirection** [comment]
It is bigger than a few steps. I did not track all the steps but kept seeing it through to the end of the course. It happens everywhere that we later do conditional action based on the Booleans stored in arrays. For each array we made, (I think there were 3? player, platform and checkpoints), the instructions use the same wording.

**Comment by jdwilkin4** [comment]
Looks like we have a PR to address the few steps I mentioned earlier.

We can review that and merge it in which will close this particular issue.
But then I can audit the project to find the other steps and create a second PR to remove the rest

**Comment by jdwilkin4** [comment]
> It is bigger than a few steps

Did a search and replace for the english files and found file more files total where the change needed to be applied.
Here is the PR to resolve that. 

https://github.com/freeCodeCamp/freeCodeCamp/pull/53248
